plasma_framework/test/helpers/utils.js

```diff
@@ -3,4 +3,48 @@ async function spentOnGas(receipt) {
     return web3.utils.toBN(tx.gasPrice).muln(receipt.gasUsed);
 }
 
-module.exports.spentOnGas = spentOnGas;
+function buildOutputGuard(outputType, outputGuardData) {
+    const hashValue = web3.utils.soliditySha3(
+        { t: 'uint256', v: outputType },
+        { t: 'bytes', v: outputGuardData },
+    );
+    const rightMostBytes20 = hashValue.substring(hashValue.length - 40, hashValue.length);
+    const paddingZeros = Array(24).fill(0).join('');
+    return `0x${paddingZeros}${rightMostBytes20}`;
```

Comment:
JS has a `String.padStart()` method for easier padding:
```
    return `0x${rightMostBytes20.padStart(64, '0')}`;
```

Reply:
today I learned! :)